gateways/http/transfer/transfer.go

```diff
@@ -35,11 +42,13 @@ func Transfer(serv *mux.Router, usecase transfer.UseCase, accountUC account.UseC
 // ShowAccount godoc
 // @Summary Make a transfer
 // @Description Transfer between accounts. The account that will make the transfer must be authenticated with a token.
-// @Param Body body TransferInput true "Body"
 // @Param Authorization header string true "Bearer Authorization Token"
 // @Accept  json
 // @Produce  json
-// @Header 201 {string} Token "request-id"
+// @Success 200 {object} Transfer
+// @Failure 400 "Failed to decode"
+// @Failure 401 "Unauthorized"
+// @Failure 500 "Unexpected internal server error"
```

Comment:
the status codes being documented are not compatible with the handler's responses here either